{"filename": "haskell-enclosed-exceptions-1.0.3-263-x86_64.pkg.tar.zst", "name": "haskell-enclosed-exceptions", "base": "haskell-enclosed-exceptions", "version": "1.0.3-263", "desc": "Catching all exceptions from within an enclosed computation", "csize": "14007", "isize": "42546", "md5sum": "d8881622114befc0b2b80c25255078fb", "sha256sum": "67034a41658aafe29fb55136173d6b3b098f9703e3a24651509633bd39cd44ff", "pgpsig": "iQIzBAABCAAdFiEEtZcfLFwQqaCMYAMPeGxj8zDXy5IFAmendZIACgkQeGxj8zDXy5LwHxAAgMwJC+TUM13j4SQgqF93flN+d1Z8gssCXGHi/UnK/EIQE+20+H4YFnklmN4BNEN1H+MpR2jkW2ofo4Mg7Fslj1jK4LyOW6ViL80raTWbmBeaymZhUy/wDQnNMJAO8flxzIT0pRY5gnkk9/NdynnXFJRqbUWL3Emg94o2KWlUEdtKYeVnK6P2+Q1gcVdnsqz1HBgicS/IoYJiM1SJfp7Bnl5g8p1JJAUd+T40jTadOkuMgqgnmlnz31nfXFZz3IZvBEcfOcd5guF3D1L5hDLCFwQjjHnFPFO2h8i/6b62kPgeESBPk9rv4B9k4qs216g9XgZb36paFmVIjzQNFA/BBpVIFhho8J8DHQHp7PeatEs2Pyd+l1qI4pT0zzMua7Y8q2CsyPnWqsqjKce3vhnX3QaRNtiDIGACQePJLWqtpRrRsjaQZXaGrmlli1XSLsIJTrcOr6bW261sv3g0AkNkCCnqQEEgV6eraSpaGB7h5pW7JzYaKmfG5AMTBAY8ZhGn38MJb88NkPx7NY3mm5tOe1yNIhxAZ+L3Bqanrt3TifPIB3ENsTFwEk8W/zH+UK2H5g8BKVKyB3CbrTtzrD3u+lneZjCQ6r8Mdzxz+S22IhYxYQpUwxDHtEKHyp5XTel/ldAKRIeKNI9TGnRmHANkLKab/lVPDKZL/kDQgMQOyJM=", "url": "https://github.com/jcristovao/enclosed-exceptions", "license": "MIT", "arch": "x86_64", "builddate": "1739027841", "packager": "Felix Yan <felixonmars@archlinux.org>", "depends": ["ghc-libs", "haskell-lifted-base", "haskell-monad-control", "haskell-transformers-base"], "makedepends": ["ghc"]}